squad-server/utils/constants.js

```diff
@@ -9,6 +9,6 @@ const SQUADJS_VERSION = JSON.parse(
 ).version;
 
 /* As set out by the terms of the license, the following should not be modified. */
-const COPYRIGHT_MESSAGE = 'Powered by SquadJS, Copyright © 2020';
+const COPYRIGHT_MESSAGE = 'Powered by SquadJS, Copyright © 2021';
```

Comment:
It might be better to do something like the following:
``` `Powered by SquadJS, Copyright © ${new Date().getFullYear()}` ```

Reply:
Is the update better?

Reply:
👍